### PR TITLE
host: Don’t try to set the bridge MTU

### PR DESCRIPTION
**Pull request by titanous**
Setting the bridge MTU errors with EINVAL if the MTU is greater than 1500, which it will be if eth0 supports jumbo frames. The bridge should automatically use the lowest MTU of all interfaces associated with it, so this setting is not necessary.

### CODE REVIEW

File: host/libvirt_lxc_backend.go [hunk 1]
```diff
@@ -76,6 +76,7 @@ type LibvirtLXCBackend struct {
 	state     *State
 	pinkerton *pinkerton.Context
 
+	ifaceMTU   int
```
Comment (lmars) [inline host/libvirt_lxc_backend.go]: Do we need this?

Comment (titanous) [inline host/libvirt_lxc_backend.go]: At some point we should pass it to containerinit, the other option is not parsing the MTU out of the file for now.

Comment (lmars) [inline host/libvirt_lxc_backend.go]: I don't feel strongly about removing it, just looked like an odd diff introducing a field which isn't used

**Comment by lmars** [comment]
LGTM

**Pull request merged**